### ISSUE TITLE
Crash Report

### ISSUE THREAD

**Issue by Kubsongames** [OpenTTD/OpenTTD#9120]
```
*** OpenTTD Crash Report ***

Crash at: Tue Apr 27 15:33:20 2021
In game date: 2095-05-19 (20)

Crash reason:
 Exception: C0000005
 Location:  000000013F754CAD
 Message:   <none>

OpenTTD version:
 Version:    1.11.1 (0)
 NewGRF ver: 1b086d64
 Bits:       64
 Endian:     little
 Dedicated:  no
 Build date: Apr 18 2021 21:16:51

Registers:
 RAX: 00000000FF000000 RBX: 000000000011CC40 RCX: 000000000011CC40 RDX: 0000000000000000
 RSI: 000000000000032A RDI: 0000000000000000 RBP: 00000000000005A0 RSP: 000000000028EFF0
 R8:  0000000000000002 R9:  0000000000000018 R10: 0000000000000002 R11: 0000000000000018
 R12: 000000000011CC40 R13: 00000000000005A0 R14: 0000000000000020 R15: 0000000000473100
 RIP: 000000013F754CAD EFLAGS: 00010202

 Bytes at instruction pointer:
 F3 AB B9 EC 88 00 00 48 8B 5C 24 30 48 83 C4 20 5F 48 FF 25 DB 64 E4 01

Stack trace:
 0000032A 00000000 0011CC40 00000000 000005A0 00000000 0011CC40 00000000
 02270AD0 00000000 3F7517FB 00000001 03C4E3F0 00000000 0000032A 00000000
 03C4E3F0 00000000 02270AD0 00000000 3FB4FE10 00000001 004C6A70 00000000
 022704C0 00000000 02270AD0 00000000 3FB4FE50 00000001 07DF33E0 00000000
 3FB4FE50 00000001 07DF33E0 00000000 0000032A 00000000 00000001 00000000
 000005A0 00000000 3F759F61 00000001 02270AD0 00000000 004C6A70 00000000
 3FB4FE10 00000001 FF000000 00000000 3FB39028 00000001 3F758BCD 00000001
 FFFFFFFF FFFFFFFF 3FB38210 00000001 02270AD0 00000000 3F75B5BD 00000001
 00000001 00000000 3F759EA9 00000001 0028F120 00000000 00000000 00000000
 3FB4FE10 00000001 3FB39028 00000001 3FB39028 00000001 3F75C0AC 00000001
 FFFFFFFF FFFFFFFF 00000000 00000000 FFFFFFFF FFFFFFFF 3FB38210 00000001
 00000000 00000000 70623034 6E612D70 00006D69 00000000 0000000A 00000000
 0000000F 00000000 4F134420 00006256 FFFFFFFF FFFFFFFF 3FB4FE10 00000001
 00000000 00000000 02270B10 00000000 02270AD0 00000000 3F75C3BF 00000001
 028D0FCE 000001B3 0028F310 00000000 02270B60 00000000 0028F310 00000000
 00000000 00000000 3F758B9F 00000001 02270B60 00000000 02270B10 00000000
 02270AD0 00000000 00000000 00000000 02270400 00000000 3F899CD9 00000001
 02270400 00000000 02270400 00000000 00000000 00000000 00000000 00000000
 000003F0 00000000 4F1344B0 00006256 0028F1C0 00000000 00000001 00000000
 0000000A 00000000 0028F1A0 00000000 00000000 00000001 00000000 00000000
 00000000 00000000 00000000 00000000 00000000 000007FE 0028F708 00000000
 3FB0F350 00000001 00000000 00000000 00000000 00000000 0028F700 00000000
 00000000 00000000 3F6C5D74 00000001 0000027F 00000000 00000000 00000000
 00000000 00000000 00001FA0 0000FFFF 00000000 00000000 00000000 00000000

Operating system:
 Name:     Windows
 Release:  6.1.7601 (Service Pack 1)
 Compiler: MSVC 1928

Configuration:
 Blitter:      40bpp-anim
 Graphics set: OpenGFX (7761)
 Language:     C:\Users\Admin\Desktop\kuba\Gry\steam\steamapps\common\OpenTTD\lang\polish.lng
 Music driver: dmusic
 Music set:    OpenMSX (7761)
 Network:      no
 Sound driver: win32
 Sound set:    OpenSFX (7758)
 Video driver: win32-opengl

Fonts:
 Small:  sprite
 Medium: sprite
 Large:  sprite
 Mono:   sprite

AI Configuration (local: 0) (current: 0):
  0: Human
  1: Human
  2: Human
  3: Human
  4: Human
  5: Human

Libraries:
 LZMA:       5.2.5
 LZO:        2.10
 PNG:        1.6.37
 Zlib:       1.2.11

Module information:
 C:\Users\Admin\Desktop\kuba\Gry\steam\steamapps\common\OpenTTD\openttd.exe handle: 000000013F4D0000 size: 8553984 crc: 9A796EBF date: 2021-04-26 18:57:13
 C:\Windows\SYSTEM32\ntdll.dll handle: 0000000077020000 size: 1671296 crc: 12C7AA90 date: 2020-01-03 03:35:05
 C:\Program Files\Avast Software\Avast\aswhook.dll handle: 000007FEFAF40000 size: 63512 crc: 68FC5CA9 date: 2021-04-26 06:01:51
 C:\Windows\system32\kernel32.dll handle: 0000000076F00000 size: 1162752 crc: 52B7958F date: 2020-01-03 03:33:39
 C:\Windows\system32\KERNELBASE.dll handle: 000007FEFCE90000 size: 408576 crc: 6945BC8E date: 2020-01-03 03:33:39
 C:\Windows\system32\OPENGL32.dll handle: 000007FEBA850000 size: 1039872 crc: 821E408D date: 2009-07-14 01:41:53
 C:\Windows\system32\msvcrt.dll handle: 000007FEFEFB0000 size: 634880 crc: 9D1B448A date: 2011-12-16 08:46:06
 C:\Windows\system32\ADVAPI32.dll handle: 000007FEFEE20000 size: 880640 crc: 796C4478 date: 2020-01-03 03:33:10
 C:\Windows\SYSTEM32\sechost.dll handle: 000007FEFD160000 size: 113664 crc: 98A6ADF4 date: 2015-05-25 18:19:09
 C:\Windows\system32\RPCRT4.dll handle: 000007FEFD570000 size: 1211392 crc: 4CAF040C date: 2020-01-03 03:33:53
 C:\Windows\system32\GDI32.dll handle: 000007FEFD4E0000 size: 405504 crc: C3BFCE30 date: 2019-11-28 03:28:53
 C:\Windows\system32\USER32.dll handle: 0000000076E00000 size: 1010688 crc: 0EAFC823 date: 2020-01-03 03:33:57
 C:\Windows\system32\LPK.dll handle: 000007FEFEFA0000 size: 41472 crc: 4257860A date: 2019-11-15 02:21:52
 C:\Windows\system32\USP10.dll handle: 000007FEFD010000 size: 806400 crc: 8CB0A4ED date: 2019-11-05 21:20:28
 C:\Windows\system32\GLU32.dll handle: 000007FEBB080000 size: 165376 crc: 3AEC0CD2 date: 2009-07-14 01:40:56
 C:\Windows\system32\DDRAW.dll handle: 000007FED5B40000 size: 569344 crc: 536DA9DD date: 2009-07-14 01:40:28
 C:\Windows\system32\DCIMAN32.dll handle: 000007FEE1490000 size: 14336 crc: 90953396 date: 2019-11-15 02:21:42
 C:\Windows\system32\SETUPAPI.dll handle: 000007FEFDBD0000 size: 1900544 crc: A364BD6A date: 2010-11-20 03:27:26
 C:\Windows\system32\CFGMGR32.dll handle: 000007FEFCE40000 size: 207872 crc: 1984C339 date: 2010-11-20 03:25:52
 C:\Windows\system32\OLEAUT32.dll handle: 000007FEFDDB0000 size: 878080 crc: 9F34D3E8 date: 2019-11-15 02:22:04
 C:\Windows\system32\ole32.dll handle: 000007FEFDE90000 size: 2072576 crc: 05ECA873 date: 2019-11-15 02:22:04
 C:\Windows\system32\DEVOBJ.dll handle: 000007FEFCDE0000 size: 93184 crc: A7541AE2 date: 2009-07-14 01:40:28
 C:\Windows\system32\dwmapi.dll handle: 000007FEFAFA0000 size: 82944 crc: F979E48B date: 2015-07-09 17:58:26
 C:\Windows\system32\WS2_32.dll handle: 000007FEFF060000 size: 296448 crc: EE1BC529 date: 2016-05-11 17:02:50
 C:\Windows\system32\NSI.dll handle: 000007FEFF050000 size: 13312 crc: C65F00AA date: 2017-08-11 06:35:01
 C:\Windows\system32\WINMM.dll handle: 000007FEFB010000 size: 217600 crc: 7BC1529F date: 2009-07-14 01:41:56
 C:\Windows\system32\IMM32.dll handle: 000007FEFD1E0000 size: 167424 crc: 41B2805A date: 2009-07-14 01:41:09
 C:\Windows\system32\MSCTF.dll handle: 000007FEFCF00000 size: 1078784 crc: E5722AE3 date: 2019-08-29 02:50:36
 C:\Windows\system32\SHELL32.dll handle: 000007FEFE090000 size: 14185984 crc: BFB2A553 date: 2019-05-25 00:04:16
 C:\Windows\system32\SHLWAPI.dll handle: 000007FEFD0E0000 size: 448512 crc: 086DF743 date: 2010-11-20 03:27:26
 C:\Users\Admin\Desktop\kuba\Gry\steam\gameoverlayrenderer64.dll handle: 000007FEBA660000 size: 1809640 crc: 8A9EFDDF date: 2021-04-12 20:03:30
 C:\Windows\system32\PSAPI.DLL handle: 00000000771D0000 size: 9216 crc: 84B5E9BA date: 2009-07-14 01:41:53
 C:\Windows\system32\api-ms-win-core-synch-l1-2-0.DLL handle: 000007FEFBD80000 size: 12024 crc: F44C9C18 date: 2019-04-12 13:05:06
 C:\Windows\system32\uxtheme.dll handle: 000007FEFAEE0000 size: 332288 crc: B0A13DF9 date: 2009-07-14 01:41:56
 C:\Windows\system32\ig4icd64.dll handle: 000007FEB00A0000 size: 13031424 crc: 93623F47 date: 2013-10-31 18:27:20
 C:\Windows\system32\CRYPTBASE.dll handle: 000007FEFC9E0000 size: 43520 crc: 739E6DEC date: 2020-01-03 03:33:20
 C:\Windows\system32\CLBCatQ.DLL handle: 000007FEFEF00000 size: 607744 crc: 939E35C9 date: 2009-07-14 01:40:15
 C:\Windows\System32\MMDevApi.dll handle: 000007FEFB430000 size: 284160 crc: CDF2EF81 date: 2009-07-14 01:41:26
 C:\Windows\System32\PROPSYS.dll handle: 000007FEFB300000 size: 1212416 crc: B83FA3BE date: 2010-11-20 03:27:24
 C:\Windows\system32\AUDIOSES.DLL handle: 000007FEFA5A0000 size: 295936 crc: B227EAC3 date: 2019-09-12 03:44:27
 C:\Windows\system32\mswsock.dll handle: 000007FEFC280000 size: 327168 crc: 2E4865A8 date: 2016-05-11 17:02:42
 C:\Windows\System32\wshtcpip.dll handle: 000007FEFBCC0000 size: 13312 crc: 5D1E2C55 date: 2009-07-14 01:41:58
 C:\Windows\system32\wdmaud.drv handle: 000007FEFA5F0000 size: 217088 crc: 12829973 date: 2010-11-20 03:24:22
 C:\Windows\system32\ksuser.dll handle: 0000000074820000 size: 5120 crc: 781141DD date: 2015-12-08 19:07:31
 C:\Windows\system32\AVRT.dll handle: 000007FEFB2F0000 size: 18432 crc: 5CE53CB7 date: 2009-07-14 01:40:05
 C:\Windows\system32\msacm32.drv handle: 000007FEFA130000 size: 25600 crc: 2B579A05 date: 2009-07-14 01:38:50
 C:\Windows\system32\MSACM32.dll handle: 000007FEFA110000 size: 83456 crc: 3D67538B date: 2009-07-14 01:41:27
 C:\Windows\system32\midimap.dll handle: 000007FEFA100000 size: 20480 crc: CB902A13 date: 2009-07-14 01:41:23
 C:\Windows\System32\dmusic.dll handle: 000007FEBA630000 size: 125952 crc: 11947851 date: 2009-07-14 01:40:31
 C:\Windows\System32\DSOUND.dll handle: 000007FEFB050000 size: 540672 crc: B6225596 date: 2009-07-14 01:40:33
 C:\Windows\System32\POWRPROF.dll handle: 000007FEFB480000 size: 167424 crc: F6D04424 date: 2009-07-14 01:41:53
 C:\Windows\system32\WINTRUST.dll handle: 0000000002080000 size: 229376 crc: 4A3D489A date: 2019-12-10 08:32:47
 C:\Windows\system32\CRYPT32.dll handle: 000007FEFCB90000 size: 1484800 crc: 8B142E5A date: 2019-12-10 08:32:28
 C:\Windows\system32\MSASN1.dll handle: 000007FEFCB40000 size: 46592 crc: 50F9CBBE date: 2010-11-20 03:27:02
 C:\Windows\System32\dmsynth.dll handle: 000007FEBA600000 size: 119296 crc: 05A882A4 date: 2009-07-14 01:40:31

---- gamelog start ----
Tick 0: new game started
Revision text changed to 1.10.3, savegame version 218, not modified, _openttd_newgrf_version = 0x1a086d64
New game mode: 1 landscape: 2
Tick 1280: game loaded
New game mode: 2 landscape: 2
Tick 1280: game loaded
New game mode: 1 landscape: 2
Tick 39224: game loaded
New game mode: 2 landscape: 2
Tick 39224: game loaded
New game mode: 1 landscape: 2
Tick 60493: settings changed
Setting changed: vehicle.never_expire_vehicles : 0 -> 1
Tick 52147: game loaded
New game mode: 2 landscape: 2
Tick 52147: game loaded
New game mode: 1 landscape: 2
Tick 39493: game loaded
New game mode: 2 landscape: 2
Tick 39493: game loaded
New game mode: 1 landscape: 2
Tick 29222: game loaded
New game mode: 2 landscape: 2
Tick 29222: game loaded
New game mode: 1 landscape: 2
Tick 29222: game loaded
Revision text changed to 1.11.1, savegame version 290, not modified, _openttd_newgrf_version = 0x1b086d64
New game mode: 0 landscape: 2
---- gamelog end ----

Recent news messages:

*** End of OpenTTD Crash Report ***

Decoded stack trace:
[00] openttd              0x000000013F754CAD
[01] openttd              0x000000013F7517FB
[02] openttd              0x000000013F759F61
[03] openttd              0x000000013F758BCD
[04] openttd              0x000000013F759EA9
[05] openttd              0x000000013F75C0AC
[06] openttd              0x000000013F75C3BF
[07] openttd              0x000000013F758B9F
[08] openttd              0x000000013F899CD9
[09] openttd              0x000000013F6D0CF8
[10] openttd              0x000000013FA37E92
[11] kernel32             0x0000000076F1556D BaseThreadInitThunk + 13
[12] ntdll                0x000000007707372D RtlUserThreadStart + 29

*** End of additional info ***
```

1.11.1

## Steps to reproduce
 Crash when start up

**Comment by glx22** [comment]
Can you try if it still happens with nightly version ? It's available in steam via openttd properties, beta.

**Comment by LordAro** [comment]
Likely duplicate of #8967

**Comment by LordAro** [comment]
Duplicate of #8967. Should be fixed in 1.11.2, released now!
